common/gas_limits.go

```diff
@@ -0,0 +1,7 @@
+package common
+
+const (
+	EVMSend = 21000
```

Comment:
Can we have a comment `// EVMSend is the gas limit required to transfer tokens on an EVM`